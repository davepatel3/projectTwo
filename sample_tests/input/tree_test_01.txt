float
allocate
-41.4618
0
allocate
10.2665
1
allocate
83.732
0
allocate
-30.3349
0
allocate
-12.83
1
allocate
-25.0967
0
allocate
63.3868
5
allocate
-15.7852
3
ref_node
7
allocate
90.716
6
allocate
8.92718
4
ref_node
7
ref_node
1
ref_node
4
allocate
-66.6509
6
allocate
75.4882
6
allocate
88.5023
12
allocate
-83.9167
12
allocate
-5.24819
0
allocate
-27.5056
15
allocate
-18.9587
10
allocate
-59.1345
14
allocate
75.841
16
ref_node
7
peek_nodes
allocate
6.4397
15
ref_node
5
allocate
-58.3122
1
quit
