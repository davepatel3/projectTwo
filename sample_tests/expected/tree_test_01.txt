allocate
1
allocate
2
allocate
3
allocate
4
allocate
5
allocate
6
allocate
7
allocate
8
ref_node
63.3868
parent: 5
children: 
allocate
9
allocate
10
ref_node
63.3868
parent: 5
children: 
ref_node
-41.4618
parent: 0
children: 2 5 
ref_node
-30.3349
parent: 0
children: 10 
allocate
11
allocate
12
allocate
13
allocate
14
allocate
15
allocate
16
allocate
17
allocate
18
allocate
19
ref_node
63.3868
parent: 5
children: 
peek_nodes
0: 0
1: -41.4618
2: 10.2665
3: 83.732
4: -30.3349
5: -12.83
6: -25.0967
7: 63.3868
8: -15.7852
9: 90.716
10: 8.92718
11: -66.6509
12: 75.4882
13: 88.5023
14: -83.9167
15: -5.24819
16: -27.5056
17: -18.9587
18: -59.1345
19: 75.841
allocate
20
ref_node
-12.83
parent: 1
children: 7 
allocate
21
quit
